packages/scenario-harness/ui/styles.css

```
@@ -0,0 +1,347 @@
+  color-scheme: dark;
+  --bg: #15171c;
+  --bg-elev: #1d2026;
+  --bg-elev-2: #232830;
+  --border: #2a2f37;
+  --text: #d8dde3;
+  --text-dim: #8d949e;
+  --accent: #4cb3ff;
+  --warn: #ffb84d;
+  --error: #ff6b6b;
+  --good: #6cdb8b;
+  --mono: ui-monospace, "Cascadia Code", "JetBrains Mono", Consolas, monospace;
```

Stylelint (17.9.0)

[error] Expected "Consolas" to be "consolas" (value-keyword-case)

(value-keyword-case)